test/functional/lua/vim_spec.lua

```diff
@@ -629,22 +629,27 @@ describe('lua stdlib', function()
   end)
 
   it('vim.wo', function()
-    eq('', funcs.luaeval "vim.bo.filetype")
     exec_lua [[
     vim.api.nvim_win_set_option(0, "cole", 2)
-    BUF = vim.api.nvim_create_buf(false, true)
-    vim.api.nvim_buf_set_option(BUF, "modifiable", false)
+    vim.cmd "split"
+    vim.api.nvim_win_set_option(0, "cole", 2)
     ]]
     eq(2, funcs.luaeval "vim.wo.cole")
     exec_lua [[
     vim.wo.conceallevel = 0
-    vim.bo[BUF].modifiable = true
     ]]
     eq(0, funcs.luaeval "vim.wo.cole")
+    eq(0, funcs.luaeval "vim.wo[0].cole")
```

Comment:
in order to test the change, shouldn't this be:

```suggestion
    eq(0, funcs.luaeval "vim.wo.cole")
```

?

Reply:
Sure, but I just beefed up the tests overall, not just for this test. Testing [0] as well helps.